### DISCUSSION TITLE
pandas_datareader - error

### DISCUSSION THREAD

**Discussion by DGUM27** [teamalgoritma/community#209] [Q&A] [answered]
Dear Bapak/Ibu,

Mohon bantuan perihal error dalam menjalankan pandas_datareader sebagaimana terlampir. 
thanks in advance. 

![Uploading Screen Shot 2022-
<img width="657" alt="Screen Shot 2022-07-16 at 12 52 31" src="https://user-images.githubusercontent.com/107378736/179341965-562d9ae7-51e2-4629-b268-679b1e0b4bb8.png">
07-16 at 12.52.12.png…]()

**Comment by idoali** [comment]
Selamat sore. Mohon maaf Pak/Bu, kami masih belum bisa melihat gambarnya, apa boleh di upload ulang gambarnya Pak/Bu?

**Comment by NabiilahArdini** [comment, marked as answer]
Halo @DGUM27 , untuk menjaga agar forum diskusi tetap berjalan kondusif, kami izin melakukan lock/close diskusi ini karena belum ada respon setelah 7 hari respon terakhir. Semoga dapat dipahami.
